editor/modes/visual-editor/style.scss

```diff
@@ -66,12 +66,17 @@
 
 .editor-visual-editor__block-controls {
 	@include animate_fade;
-	position: absolute;
-	bottom: 100%;
-	margin-bottom: -4px;
-	left: #{ $block-padding * 2 + $block-mover-margin };	// compensate for mover, and indent same as padding
 	display: flex;
+	position: sticky;
 	z-index: 1;
+	margin-top: -56px;
```

Comment:
Yep, was looking for a way to replace that with variables, but couldn't immediately put it together and I had to relocate. Definitely will, though.